OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -0,0 +1,440 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
```

Comment:
I think this should be `OWML.ModHelper.Input`?

Reply:
done

Reply:
btw, it screwed up UnityEngine.Input somehow

Reply:
that is I can't just write Input.GetKey and have to write UnityEngine.Input.GetKey intead